tango/integrations/datasets/__init__.py

```diff
@@ -72,22 +101,65 @@ class LoadDataset(Step):
 
         Registered as a :class:`~tango.step.Step` under the name "datasets::load".
 
+    .. important::
+
+        If you are loading an :class:`~datasets.IterableDataset` or :class:`~datasets.IterableDatasetDict`
+        you need to use the :class:`LoadStreamingDataset` step instead.
+
     """
 
     DETERMINISTIC = True
     VERSION = "001"
-    CACHEABLE = False  # These are already cached by huggingface.
```

Comment:
To be super clear, we want to do this because even though huggingface caches datasets, their `load_dataset()` function is still sometimes slow enough to make this worth it?

Reply:
I feel like that may be worth pointing out somewhere in a doc or a comment.

Reply:
Yes, I'll add a comment.

Reply:
32face0